openhands/cli/gui_launcher.py

```diff
@@ -1,219 +0,0 @@
-"""GUI launcher for OpenHands CLI."""
-
-import os
-import shutil
-import subprocess
-import sys
-from pathlib import Path
-
-from prompt_toolkit import print_formatted_text
-from prompt_toolkit.formatted_text import HTML
-
-from openhands import __version__
-
-
-def _format_docker_command_for_logging(cmd: list[str]) -> str:
-    """Format a Docker command for logging with grey color.
-
-    Args:
-        cmd (list[str]): The Docker command as a list of strings
-
-    Returns:
-        str: The formatted command string in grey HTML color
-    """
-    cmd_str = ' '.join(cmd)
-    return f'<grey>Running Docker command: {cmd_str}</grey>'
-
-
-def check_docker_requirements() -> bool:
-    """Check if Docker is installed and running.
-
-    Returns:
-        bool: True if Docker is available and running, False otherwise.
-    """
-    # Check if Docker is installed
-    if not shutil.which('docker'):
-        print_formatted_text(
-            HTML('<ansired>❌ Docker is not installed or not in PATH.</ansired>')
-        )
-        print_formatted_text(
-            HTML(
-                '<grey>Please install Docker first: https://docs.docker.com/get-docker/</grey>'
-            )
-        )
-        return False
-
-    # Check if Docker daemon is running
-    try:
-        result = subprocess.run(
-            ['docker', 'info'], capture_output=True, text=True, timeout=10
-        )
-        if result.returncode != 0:
-            print_formatted_text(
-                HTML('<ansired>❌ Docker daemon is not running.</ansired>')
-            )
-            print_formatted_text(
-                HTML('<grey>Please start Docker and try again.</grey>')
-            )
-            return False
-    except (subprocess.TimeoutExpired, subprocess.SubprocessError) as e:
-        print_formatted_text(
-            HTML('<ansired>❌ Failed to check Docker status.</ansired>')
-        )
-        print_formatted_text(HTML(f'<grey>Error: {e}</grey>'))
-        return False
-
-    return True
-
-
-def ensure_config_dir_exists() -> Path:
-    """Ensure the OpenHands configuration directory exists and return its path."""
-    config_dir = Path.home() / '.openhands'
-    config_dir.mkdir(exist_ok=True)
-    return config_dir
-
-
-def launch_gui_server(mount_cwd: bool = False, gpu: bool = False) -> None:
```

Comment:
V1 CLI no longer supports this entry point since its a standalone project. Thoughts on backwards compatibility here?

Reply:
hmm... maybe we can leave it for now, i imagine we will update the quick start for v1 completely..